rgw/v2/lib/sync_status.py

```diff
@@ -25,6 +25,8 @@ def sync_status(retry=25, delay=60, ssh_con=None):
     else:
```

Comment:
@ckulal , could you check error output and log.error() it as well please
https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/utils/utils.py#L110

Reply:
done